packages/flutter/lib/src/rendering/table.dart

```diff
@@ -347,7 +347,10 @@ enum TableCellVerticalAlignment {
 
   /// Cells with this alignment are sized to be as tall as the row, then made to fit the row.
   /// If all the cells have this alignment, then the row will have zero height.
-  fill
+  fill,
+
+  /// Cells with this alignment are sized to be the same height as the tallest cell in the row.
+  intrisicHeight
```

Comment:
Adding an enum value is a breaking change since it will break any switch statements on TableCellVerticalAlignment that are not exhaustive because we are adding a new one here.

That does not mean we cannot land the change, I'll just need to investigate to find out what the impact would be first.